apps/journeys-admin/src/components/TemplateGalleryPageList/MobileTemplateList/MobileTemplateListRow.spec.tsx

```
@@ -0,0 +1,63 @@
+import { MockedProvider } from '@apollo/client/testing'
+import { render, screen } from '@testing-library/react'
+import { SnackbarProvider } from 'notistack'
+
+import '../../../../test/i18n'
+
+import { ThemeProvider } from '../../ThemeProvider'
+import {
+  customizableTemplateJourney,
+  customizableWebsiteTemplateJourney,
+  defaultJourney
+} from '../../JourneyList/journeyListData'
```

ESLint

[error] `../../JourneyList/journeyListData` import should occur before import of `../../ThemeProvider`

(import/order)